minecoins.lua

```diff
@@ -1,33 +1,7 @@
 -- Node definitions
-minetest.register_node("bitchange:minecoin_in_ground", {
-	description = "you hacker!",
-	tiles = { "default_stone.png^bitchange_minecoin_in_ground.png" },
-	is_ground_content = true,
-	groups = {cracky=2, not_in_creative_inventory=1},
-	sounds = default.node_sound_stone_defaults(),
-	drop = {
-		max_items = 2,
-		items = {
-			{items = {"bitchange:minecoin"}, rarity = 2 },
-			{items = {"bitchange:minecoin 3"} }
-		}
-	},
-})
+minetest.register_alias("bitchange:minecoin_in_ground", "default:stone_with_gold")
```

Comment:
It would be nice to have a notification so that people don't wonder what happened to it.
```Lua
-- Legacy
if bitchange.enable_generation then
	minetest.log("warning", "[bitchange] Ores will not be generated any more. Remove the setting " ..
		"'enable_generation' from your bitchange configuration to suppress this warning.")
end
```